### ISSUE TITLE
Demonstration with telegram bot

### ISSUE THREAD

**Issue by edxu96** [pasty-dev/cmdict#104]
https://github.com/python-telegram-bot/python-telegram-bot or https://docs.pyrogram.org .

**Comment by edxu96** [comment]
I created a telegram bot named `cmdicd_bot`. You can start a conversion with `@cmdict_bot` now.

Just tried to run [this `echobot.py` example](https://github.com/python-telegram-bot/python-telegram-bot/blob/master/examples/echobot.py) on my laptop. It works.

---

The official bot of `telegram` for managing user's bots is a very good example, like there is a menu button for different options:

<img width="815" alt="Screenshot 2022-05-01 at 20 29 37" src="https://user-images.githubusercontent.com/19307850/166159501-021216ca-e20c-493f-b080-96c85d474599.png">

Basically, we can easily have an `cmdict` web/phone/laptop application.

@zequnyu I think this is our next step. I can write another small Python script, and we run this script in AWS. Any new user can try or even use `cmdict` on a daily basis in telegram.

**Comment by edxu96** [comment]
We can host in a Mac in https://www.macstadium.com/opensource.

**Comment by edxu96** [comment]
The development is ongoing in https://github.com/pasty-dev/cmdict_bot.
